### DISCUSSION TITLE
ScriptReference/Application.Quit

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#195] [Notes]
# ScriptReference/Application.Quit

https://docs.unity3d.com/ScriptReference/Application.Quit.html

**Comment by mvi** [comment]
Note that while the documentation and guidelines discourage usage of `Application.Quit()` for released games, it does work correctly on iOS. This is particularly useful for developer code that may need to force quit the game.
